### DISCUSSION TITLE
[Art competition] Semantics

### DISCUSSION THREAD

**Discussion by park-sunho** [prosyslab-classroom/cs424-program-reasoning#250] [Art competition]
![DALL·E 2022-11-29 19 53 32 - wide road in the middle, skyscrapers at each side of the road, no people, devastated city](https://user-images.githubusercontent.com/79883007/204510901-6cc28341-9d19-49de-bc12-50f0e3b623bc.png)

## Short Description
For me, [*Semantics*] means [*wide road in the middle, skyscrapers at each side of the road, no people, devastated city*]".

## Long Description (up to 3 sentences)
Without human, everything in modern city becomes meaningless as depicted in the picture. Likewise, without semantics of a language, everything we've talked in the class including programming, program verification, and program synthesis become meaningless. In this sense, we can say the role of semantics in programming is similar to the role of human in modern city.

**Comment by KihongHeo** [comment]
Oh, you emphasize human (semantics) by NOT drawing human (semantics)? 烘雲托月?

**Comment by park-sunho** [comment]
Yes. I could only come up with this idea to emphasize human effectively. I'm happy that you understood my intention correctly.
